addons/portal/static/src/scss/portal.scss

```diff
@@ -319,6 +324,18 @@ img.o_we_custom_image {
 }
 
 // Portal specific
+
+// ===  Page custom bg (eg. 'Invoice') ===
+// This could have been achieved easily using a class (eg. bg-200), but we
+// don't want to force the design because of themes.
```

Comment:
Not sure to understand the reason, why not using bg-200 as you are suggesting? Themes can still change the color of bg-200. Or maybe even using bg-light ? (which does not imply a gray color)

Reply:
@stefanorigano You ignored this comment. I already did this: https://github.com/odoo-dev/odoo/commit/cd8510862c00e049bcf64ff09e051649e06cf4b5 but JKE is asking the very reason of this. Why is this supposed to be gray sometimes and sometimes not ?

Reply:
@qsm-odoo , sorry, I missed the comment not ignored :)

I didn't use the class because:
* bg-200 was too much colorful and I needed something different.
* themes can change the `$gray-200` value, true, but it's not scoped for portal, it will change everywhere.
* In my opinion add `$o-portal-bg-color: $mycolor` in themes it's simpler and easier to maintain  than xpath to replace the class. 

These are my reasons, but I don't know very well how website interacts with themes anymore.. I noticed that a lot of stuffs changed since the last time I worked on website apps, so maybe I'm wrong.

> Why is this supposed to be gray sometimes and sometimes not ?

We wanted to highlight the fact that you're visualizing a single document (invoice, sale order..), so I used a subtle different bg.

cc: @JKE-be